communication/youtube/youtube-guidelines.md

```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
 
 ### Uploading Guidelines for Collaborators
 
-**NOTE:** If you're using a G Suite account you may need to [loosen the
-permissions in your YouTube settings]. If you have any questions reach out to
-the [YouTube admins] or [SIG Contributor Experience].
+**NOTE:** If you're using a Google Workspace account (formerly known as G Suite)
+you may need to [update the permissions in your YouTube settings][]. If you have
+any questions, reach out to the [YouTube admins][] or
+[SIG Contributor Experience][]. You may need to reach out to someone at your
+organization if you do not have access to Google Workspace Admin permissions.
 
-**NOTE:** Both public and private steering meeting recordings should be made public.
+**NOTE:** Both public and private steering meeting recordings should be made
+public.
 
 With collaboration comes great responsibility. Playlist collaborators in the
 community must use it responsibly and are subject to the following guidelines:
 
-- Group leaders or other appropriate point(s) of contact are the primary
-  managers for the playlist, once collaboration is configured. YouTube admins
-  should **only** be contacted if the issue cannot be resolved by one of the
-  playlist owners.
+-   Group leaders or other appropriate point(s) of contact are the primary
+    managers for the playlist, once collaboration is configured. YouTube admins
+    should **only** be contacted if the issue cannot be resolved by one of the
+    playlist owners.
+
+-   Upload responsibilities belong to the group leaders or other appropriate
+    contacts. YouTube admins should **only** be contacted if the issue cannot be
+    resolved by the playlist owners.
+
+-   Please post only related content, for example meeting recordings, in the
+    appropriate playlists.
+    -   Posting of any exceedingly inappropriate content (i.e. NSFW content)
+        will result in ***immediate*** suspension of privileges.
+
+-   All posted videos should use the naming convention: \<Name of Playlist's
+    Group\> YYYYMMDD
+    -   **Example:** SIG Service Catalog 20161129
+
+-   Playlists should be organized chronologically for ease of use. This can be
+    done by updating the default ordering of the Playlist:
+    -   From within the Playlist settings, click on the **Basic** Tab.
+    -   From the **Ordering** dropdown select "Date added (newest)".
+    -   Save the changes and the order should automatically be updated.
+
+-   Please do not remove any already-published content from the playlists
+    without checking with the YouTube admins.
+
+-   For any small issues that arise, for example improper naming or ordering,
+    you may be asked by the YouTube admins to attempt to resolve the issue
+    yourself.
+
+-   Any egregious or habitual violations (3 or more per quarter) of the above
```

Comment:
I think this should be discussed more. Audit trails do not exist in YouTube so enforcement without concrete proof is setting us up for a bad time

Reply:
Yeah, this isn't followed well and isn't enforced. There's too much overhead in it already. I do think we should tackle that in a separate issue/PR - this one is style guide changes.